<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QMainWindow" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="loginBackgroundFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="loginViewWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>561</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>登入</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginPwdInput">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QComboBox" name="loginIdInput">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="duplicatesEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="autoLoginCheck">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>自动登入</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="remPwdCheck">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logonButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#ffffff;&quot;&gt;MyQQ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>159</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;当前服务器状态：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLable">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;离线&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="backgroud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/background.gif);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="minButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelLoginButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Algerian&quot;;
cancelLoginButton{border:1px solid white;background:transparent;}</string>
     </property>
     <property name="text">
      <string>取消登入</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>backgroud</zorder>
    <zorder>loginViewWidget</zorder>
    <zorder>title</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>minButton</zorder>
    <zorder>closeButton</zorder>
    <zorder>cancelLoginButton</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
